docs/source/releasenotes.rst

```diff
@@ -9,58 +9,69 @@ Release notes
 `Released <https://pypi.org/project/rpaframework/#history>`_
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-22.2.1 - 09 Mar 2023
+22.2.2 - 10 Mar 2023
 --------------------
 
-- Library **RPA.Calendar**:
+- Library **RPA.Browser.Selenium** (:pr:`859`):
+
+  - Fixes ``Set Download Directory`` keyword: functionality, docs, examples.
+  - Improves Browser Locators functionality. (referenced by `alias:`)
 
-  Documentation fixes to various keywords.
+22.2.1 - 09 Mar 2023
+--------------------
+
+- Library **RPA.Calendar**: Documentation fixes in various keywords.
 
 22.2.0 - 09 Mar 2023
 --------------------
 
 - New library **RPA.Calendar** (:pr:`694`):
 
-  - Read library documentation for full list of keywords
-  - Supports holiday and business day related calculations
+  - Supports holiday and business day related calculations.
+  - Read library documentation for a full list of keywords.
 
 - Library **RPA.Database**:
 
-  - Change how `psycopg2` module handles the ``Call Stored Procedure`` keyword
-  - Set on the `autocommit` parameter use with ``Connect To Database`` keyword when using `psycopg2` module
-  - Remove **INFO** level logs from ``Connect To Database`` keyword
+  - Changes how `psycopg2` module handles the ``Call Stored Procedure`` keyword.
+  - Sets on the `autocommit` parameter use with ``Connect To Database`` keyword when
+    using `psycopg2` module.
+  - Removes **INFO** level logs from ``Connect To Database`` keyword. (privacy)
 
 - Library **RPA.Outlook.Application** (:pr:`878`):
 
-  - Add possibility set **CC** and **BCC** recipients with ``Send Email`` and ``Send Message`` keywords
-
+  - Adds possibility to set **CC** and **BCC** recipients with ``Send Email`` and
+    ``Send Message`` keywords.
 
 08 Mar 2023
 -----------
-- Library **RPA.Assistant** (``rpaframework-assistant`` **2.1.0**):
 
-    - Features:
-
-        - Add ``Set Title`` keyword. Can be used to set title of assistant when it is running.
-
-        - Add ``Open Row`` and ``Close Row`` keywords. Used to layout elements into rows.
-        - Add ``Open Column`` and ``Close Column`` keywords. Used to layout elements into columns.
-        - Add ``Open Navbar`` and ``Close Navbar`` keywords. Can be used to create an always visible top bar for a dialog.
-
-        - Add ``Open Stack`` and ``Close Stack`` keywords. Can be used to position elements manually.
-        - Add ``Open Container`` and ``Close Container`` keywords. Useful for styling or placing single elements.
-
```

Comment:
@xylix These kind of spaces doesn't make any difference in the rendered bullet-points, isn't that?